x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -494,6 +528,63 @@ export const EndpointList = () => {
     return endpointsExist && !patternsError;
   }, [endpointsExist, patternsError]);
 
+  const transformFailedCalloutDescription = useMemo(
```

Comment:
As this is not depending on any state/props variable, you can get rid of useMemo here.

Reply:
you're right, this has no calculations so useMemo is probably more expensive than just rendering so will remove in new commit.